src/test/java/uk/gov/pay/webhooks/healthcheck/HealthCheckResourceIT.java

```diff
@@ -5,26 +5,44 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 import uk.gov.pay.extension.AppWithPostgresAndSqsExtension;
+import uk.gov.pay.rule.PostgresTestDocker;
 
 import static org.hamcrest.Matchers.equalTo;
 
-public class HealthCheckResourceIT {
+class HealthCheckResourceIT {
     @RegisterExtension
     public static AppWithPostgresAndSqsExtension app = new AppWithPostgresAndSqsExtension();
 
     @Test
-    public void HealthCheckIsHealthyTest(){
+    void HealthCheckIsHealthyTest(){
         RestAssured.given().port(app.getAppRule().getLocalPort())
                 .contentType(ContentType.JSON)
                 .when()
                 .accept(ContentType.JSON)
                 .get("healthcheck")
                 .then()
                 .statusCode(200)
-                .body("[0].database.healthy", equalTo(true))
-                .body("[1].deadlocks.healthy", equalTo(true))
-                .body("[2].hibernate.healthy", equalTo(true))
-                .body("[3].ping.healthy", equalTo(true));
+                .body("database.healthy", equalTo(true))
+                .body("deadlocks.healthy", equalTo(true))
+                .body("hibernate.healthy", equalTo(true))
+                .body("ping.healthy", equalTo(true))
+                .body("sqsQueue", equalTo(true));
     }
 
+    @Test
+    void healthCheckShouldReturn503() throws InterruptedException {
```

Comment:
Could stop SQS too and check that healthy is false for SQS

Reply:
👍